### PR TITLE
check: Suggest usage of `repair packs` if pack files are damaged

### PR DESCRIPTION
**Pull request by MichaelEischer**
What does this PR change? What problem does it solve?
-----------------------------------------------------
`check --read-data` currently only provides instructions on how to repair a repository, if the damage was likely caused by the compression bug in restic 0.16.0. This PR lets `check` also print the instructions for other types of pack damages. It also removes the feature flag from `repair packs`.

Was the change previously discussed in an issue or on the forum?
----------------------------------------------------------------
No. 

Checklist
---------

- [x] I have read the [contribution guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches).
- [x] I have [enabled maintainer edits](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added tests for all code changes.
- [x] I have added documentation for relevant changes (in the manual).
- [x] There's a new file in `changelog/unreleased/` that describes the changes for our users (see [template](https://github.com/restic/restic/blob/master/changelog/TEMPLATE)).
- [x] I have run `gofmt` on the code in all commits.
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits).
- [x] I'm done! This pull request is ready for review.

### CODE REVIEW

File: changelog/unreleased/pull-4644 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Enhancement: `repair packs` works in more cases
```
Comment (rawtaz) [inline changelog/unreleased/pull-4644:1]:
~~~
Enhancement: Improve `repair packs` command
~~~

File: doc/077_troubleshooting.rst [hunk 2]
```diff
@@ -76,6 +76,10 @@ Similarly, if a repository is repeatedly damaged, please open an `issue on Githu
 somewhere. Please include the check output and additional information that might
 help locate the problem.
 
+If ``check`` detects damaged pack files, it will show instructions on how to repair
+them using the ``repair pack`` command. Use that command instead of the "repair the
```
Comment (rawtaz) [inline doc/077_troubleshooting.rst:80]: s/repair the/Repair the/

File: doc/077_troubleshooting.rst [hunk 3]
```diff
@@ -76,6 +76,10 @@ Similarly, if a repository is repeatedly damaged, please open an `issue on Githu
...
+index" section from this guide.
```
Comment (rawtaz) [inline doc/077_troubleshooting.rst:81]: s/from/in/

**Review by rawtaz — CHANGES_REQUESTED** [review]
🦄

File: changelog/unreleased/pull-4644 (hunk 1 above)
Comment (MichaelEischer) [reply changelog/unreleased/pull-4644]: Thanks! I've applied all suggestions.

**Review by rawtaz — APPROVED** [review]
LGTM!

**Pull request merged**